org.elastic.jdt.ls.core/src/org/elastic/jdt/ls/core/internal/SecurityPermissionHelper.java

```diff
@@ -0,0 +1,115 @@
+package org.elastic.jdt.ls.core.internal;
+
+import java.io.File;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import org.eclipse.core.runtime.Platform;
+import org.eclipse.jdt.ls.core.internal.Environment;
+import org.osgi.framework.BundleContext;
+import org.osgi.framework.ServiceReference;
+import org.osgi.service.condpermadmin.ConditionalPermissionAdmin;
+import org.osgi.service.condpermadmin.ConditionalPermissionInfo;
+import org.osgi.service.condpermadmin.ConditionalPermissionUpdate;
+
+public final class SecurityPermissionHelper {
+	
+	private static final String CLIENT_PORT = "CLIENT_PORT";
+	private static final String CLIENT_HOST = "CLIENT_HOST";
+	// Extra white host for repository, separated by commas
+	private static final String EXTRA_WHITELIST_HOST = "EXTRA_WHITELIST_HOST";
+	
+	// Default repository white list
+	private static final List<String> DEFAULT_HOST_WHITELIST = Arrays.asList(
```

Comment:
Is the list designed to be immutable? Maybe try to wrap it with Collections.unmodifiableList()

Reply:
Will do